turbo.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "pipeline": {
+  "envMode": "loose",
```

Comment:
Turbo v2 now defaults to strict mode for the env mode. We should set this to loose until we've refactored our configs to cater for the new default behaviour - https://turborepo.com/docs/reference/configuration#envmode